### PR TITLE
Build errors

### PR DESCRIPTION
**Pull request by johnnolan**
### Description of the Change

Fix image logo size and broken speaker links

### Benefits

Links and image are no longer broken

### Applicable Issues

Depends on #890

### CODE REVIEW

File: pages/pastconferences/pastspeakers/pastspeakergrid.js [hunk 1]
```diff
@@ -28,111 +28,111 @@ export default function Speaker2021Component() {
         <SpeakerGridItemComponent
           speakername="Tania Allard"
           pic="/static/speakers/2021/Tania_A.jpg"
-          link="../2021/tania-allard"
```
Comment (wordshaker) [inline pages/pastconferences/pastspeakers/pastspeakergrid.js:31]: I just checked on the production website. These links work currently, they aren't broken....?

**Comment by johnnolan** [comment]
@wordshaker saw you merged and reverted the warnings PR. This one does the same but sets the logo height and width properly

**Review by wordshaker — COMMENTED** [review]
Running locally now to give it a once over now

**Comment by wordshaker** [comment]
Styling still broken my dude:

image

**Comment by johnnolan** [comment]
This should now be working including a font loading issue fixed

image

**Review by wordshaker — APPROVED** [review]

**Comment by wordshaker** [comment]
I'm happy, Merge away

**Pull request merged**